### PR TITLE
Update Promscale resource recommendation

### PR DESCRIPTION
**Pull request by arajkumar**
# Description

Updated resource recommendation based on recent benchmarking efforts, also included additional columns like shared buffers and active series for more granular configuration.

Signed-off-by: Arunprasad Rajkumar <ar.arunprasad@gmail.com>

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by Rajakavitha1 — APPROVED** [review]

File: promscale/guides/resource-recomm.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ remote_write:
   remote_timeout: 100s
```
Comment (arajkumar) [inline promscale/guides/resource-recomm.md:35]: I'm not sure why this file exists, may be we should remove this? @VineethReddy02

Comment (VineethReddy02) [reply promscale/guides/resource-recomm.md]: Looks like we didn't drop this file when we moved it from `guides` to `recommendations`. @arajkumar please drop the file if there is no reference for the same. 

Also, can add information on what are shared buffers and how users can allocate it?

Comment (arajkumar) [reply promscale/guides/resource-recomm.md]: Sure, I'm adding it.

File: promscale/recommendations/resource-recomm.md [hunk 2]
```diff
@@ -45,23 +45,31 @@ remote_write:
...
-|200k samples/sec|8|4.5 GB|16|64 GB|8|
+|Ingestion Rate|Active Series|Connector CPU|Connector Memory|DB CPU|DB Memory|DB connections|DB Shared Buffers|
+|-|-|-|-|-|-|-|-|
+|10k samples/sec|500K|0.5|2 GB|1|12 GB|1|8 GB|
+|50k samples/sec|2M|2|8 GB|4|32 GB|2|16 GB|
+|100k samples/sec|3M|2|8 GB|8|64 GB|4|24 GB|
+|200k samples/sec|5M|4|16 GB|16|64 GB|8|40 GB|
+
+<highlight type="note">
+You can use `shared_buffers` PostgreSQL configuration to set `DB Shared Buffers`.
```
Comment (VineethReddy02) [inline promscale/recommendations/resource-recomm.md:56]: @arajkumar how about `You should use `shared_buffers` in PostgreSQL configuration to set `DB Shared Buffers`.`

**Review by VineethReddy02 — COMMENTED** [review]
nit

**Comment by VineethReddy02** [comment]
@Rajakavitha1 re-requested a review from you as the docs have been slightly updated.

File: promscale/recommendations/config-recomm.md [hunk 3]
```diff
@@ -16,20 +16,18 @@ there are a few other PostgreSQL parameters that you might like to adjust:
...
-  delays.
-*   `bgwriter_lru_maxpages=100000` - increase the number of pages a background
-  writer can handle, to make it more efficient.
 *   `max_wal_size` - set it to a high enough value so that the checkpoint is triggered
   by the timeout setting, much before the `max_wal_size` is reached.
 *   `synchronous_commit=off` - this does not cause data corruption or
   inconsistency. However, in case of a crash, some of the most recent data points could be
   lost. For a monitoring observability use case, it's a reasonable trade-off to
   increase ingest performance.
+* `shared_buffers` - set it to at least 50% of available memory, we tried to estimate it for metrics ingestion,  see [Promscale Resource recommendations guide][promscale-resource-recommendations].
```
Comment (Rajakavitha1) [inline promscale/recommendations/config-recomm.md:25]:
```suggestion
* `shared_buffers` - set it to at least 50% of available memory and estimated based on metrics ingestion,  see [Promscale Resource recommendations guide][promscale-resource-recommendations].
```

File: promscale/recommendations/resource-recomm.md (hunk 2 above)
Comment (Rajakavitha1) [inline promscale/recommendations/resource-recomm.md:56]:
```suggestion
Use `shared_buffers` in PostgreSQL configuration to set `DB Shared Buffers`.
```

File: promscale/recommendations/resource-recomm.md [hunk 4]
```diff
@@ -45,23 +45,31 @@ remote_write:
...
-|200k samples/sec|90 days|~40 GB|~420 GB|~4 TB|-|
+|Ingest rate|Active Series|Retention|Disk size per day|Uncompressed buffer|Total disk size|WAL size|
+|-|-|-|-|-|-|-|
+|10k samples/sec|500K|90 days|~2 GB|~21 GB|~200 GB|6 GB|
+|50k samples/sec|2M|90 days|~10 GB|~105 GB|~1 TB|25 GB|
+|100k samples/sec|3M|90 days|~20 GB|~210 GB|~2 TB|66 GB|
+|200k samples/sec|5M|90 days|~40 GB|~420 GB|~4 TB|96 GB|
+
+<highlight type="note">
+You can use `max_wal_size` PostgreSQL configuration to set desired `WAL size`.
```
Comment (Rajakavitha1) [inline promscale/recommendations/resource-recomm.md:71]:
```suggestion
Use `max_wal_size` in PostgreSQL configuration to set desired `WAL size`.
```

**Review by Loquacity — APPROVED** [review]

**Pull request closed without merging**